### DISCUSSION TITLE
How to initialise the A2DP Sink as controller

### DISCUSSION THREAD

**Discussion by AdmiralAki99** [pschatzmann/ESP32-A2DP#414] [Q&A] [answered]
Hi, I have gone through the documentation for this library and I wanted to know if there was a way I can configure the A2DP Sink as a controller rather than a receiver/speaker for audio? I’ve tried to implement the AVRCP profile from the espressif-idf but cannot get it to detect. This library is the only one that works so far. Thanks!

**Comment by pschatzmann** [comment, marked as answer]
The answer is still no!
https://github.com/pschatzmann/ESP32-A2DP/discussions/350
